src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -1,113 +1,194 @@
+        || $(element).hasClass('BRpageview'); /* empty page placeholder */
+    };
+
+    /**
+     * Main hook into toggle functionality
+     * This is the only function that should be called by the event handlers
+     *
+     * @param { object } br - BookReader instance
+     * @param { object } e - JS event object
+     * @param { boolean } atBookCenter - optional
+     */
+    var toggleRouter = function toggleRouter (br, e, atBookCenter) {
+      var isValidClickArea = atBookCenter ? isCenterClick(e) : isBackground(e.target);
 
-      var brContainer = document.querySelector('.BRcontainer') || {};
-      var mainBRWrapper = document.querySelector('.BookReader') || {};
-      var firstChild = brContainer.firstChild;
-  
-      if (firstChild) {
-        firstChild.addEventListener('click', function(e) {
-          toggleMenuIfCenterClick(br, e);
-        }, true);
-      }
+      if (isValidClickArea) {
+        toggleNav(br, atBookCenter);
 
-      var toggleAtBackgroundClick = function toggleAtBackgroundClick(e) {
-        var isBackground = $(event.target).hasClass('BookReader')
-          || $(event.target).hasClass('BRcontainer') /* main black theatre */
-          || $(event.target).hasClass('BRemptypage') /* empty page placeholder */
-          || $(event.target).hasClass('BRpageview'); /* empty page placeholder */
-        if (isBackground) {
-          toggleNav(br);
+        if (atBookCenter) {
+          e.stopPropagation(); // don't turn the page. this takes prescendence
         }
-      };
-      mainBRWrapper.addEventListener('click', toggleAtBackgroundClick, true);
+      }
     }
 
-    BookReader.prototype.initMenuToggle = function brInitMenuToggle(e) {
+    /**
+     * background click event handler
+     * @param { object } br - BookReader instance
+     * @param { object } e - JS event object
+     */
+    function onBackgroundClick(br, e) {
+      toggleRouter(br, e);
+    }
+
+    /**
+     * actual book container click event handler
+     *
+     * @param { object } br - BookReader instance
+     * @param { object } e - JS event object
+     */
+    function onBookClick(br, e) {
+
+      var atBookCenter = true;
+      toggleRouter(br, e, atBookCenter);
+    }
+
+    /**
+     * attaches click handlers to background & book
+     * @param { object } br - BookReader instance
+     */
+    function registerClickHandlers(br) {
+      var background = document.querySelector('.BookReader') || {};
+      background.addEventListener('click', onBackgroundClick.bind(null, br), { capture: true, passive: true });
+
+      var desk = document.querySelector('.BRcontainer') || {};
+      var book = desk.firstChild;
+
+      if (book) {
+        book.addEventListener('click', onBookClick.bind(null, br), true);
+      }
+    }
+
+    /**
+     * Install menu toggle
+     * attaches event handlers, sets up DOM on load
+     */
+    BookReader.prototype.installMenuToggle = function installMenuToggle(e) {
```

Comment:
Does this method need to be public on the `BookReader.prototype` or can it be "internal" like `registerClickHandlers` is?

Reply:
I was patterning it with the other plugin code, where the initializer is bound to the object. 
Should we start changing that behavior?

Reply:
Ah hmm.. I guess in my mind these types of methods don't really need to be used by the consumer and attached to the "public/prototype" interface. They're typically used internally by the initializer so I tend to not expose anything publicly if they don't need to access it.

Reply:
ya, for sure. let me update.